grafana-plugin/src/pages/settings/tabs/ChatOps/ChatOps.tsx

```diff
@@ -150,6 +150,14 @@ const Tabs = (props: TabsProps) => {
           </HorizontalGroup>
         </VerticalTab>
       )}
+      {store.hasFeature(AppFeature.Mattermost) && (
+        <VerticalTab id={ChatOpsTab.Mattermost}>
+          <HorizontalGroup>
+            <Icon name="message" />
+            Mattermost
+          </HorizontalGroup>
+        </VerticalTab>
+      )}
```

Comment:
```suggestion
      {store.hasFeature(AppFeature.Mattermost) && (
        <VerticalTab id={ChatOpsTab.Mattermost}>
          <Stack>
            <Icon name="message" />
            Mattermost
          </Stack>
        </VerticalTab>
      )}
```
(fixing lint error; also note that this block is duplicated above)